core/src/test/java/org/apache/calcite/avatica/AvaticaResultSetConversionsTest.java

```diff
@@ -215,15 +270,18 @@ public TestMetaImpl(AvaticaConnection connection) {
       List<Object> row = Collections.<Object>singletonList(
           new Object[] {
               true, (byte) 1, (short) 2, 3, 4L, 5.0f, 6.0d, "testvalue",
-              new Date(1476130718123L), new Time(1476130718123L),
-              new Timestamp(1476130718123L),
+              new Date(DST_INSTANT), new Time(DST_INSTANT),
```

Comment:
Note:

- `DST_INSTANT = 1476130718123`
- `STANDARD_INSTANT = 1479123123242`
- `VALID_TIME = 1476123`
- `OVERFLOW_TIME = 147912242`

So this is not changing any of the test values; it's just giving them names so that equivalence is more apparent throughout the file.

Reply:
When I merge I might split out a 'refactor' commit that does not add or change tests but just moves literals into constant fields.